src/lib/formats/flopimg.cpp

```diff
@@ -250,26 +224,16 @@ floperr_t floppy_create(void *fp, const struct io_procs *procs, const struct Flo
 	/* if this format expects creation parameters and none were specified, create some */
 	if (!parameters && format->param_guidelines)
 	{
-		alloc_resolution = std::make_unique<util::option_resolution>(floppy_option_guide, format->param_guidelines);
+		alloc_resolution = std::make_unique<util::option_resolution>(floppy_option_guide);
 		if (!alloc_resolution)
```

Comment:
This check for null won't work - `std::make_unique` never returns a null pointer, it will throw a bad allocation exception.  This must have been introduced in a previous round of refactoring, but you should still notice/fix this stuff when changing surrounding lines.